waspc/data/Generator/templates/sdk/server/utils.ts

```diff
@@ -40,33 +40,3 @@ async (req: RequestWithExtraFields, res: Response, next: NextFunction) => {
 }
 
 export const sleep = (ms: number): Promise<unknown> => new Promise((r) => setTimeout(r, ms))
-
-export function getDirPathFromFileUrl(fileUrl: string): string {
```

Comment:
Since this code relied on importing from a specific directory e.g. `config` and bundling gets rid of the folder structure and bundles everything in one file that we can't use this method anymore.

Reply:
Yes, we used the dynamic imports based on folder structure, but now there is not folder structure we can rely on. All of the code is bundled together. Not a big deal, we used it only for auth provider imports.